sdcm/cluster_aws.py

```diff
@@ -457,14 +459,20 @@ def get_aws_termination_notification(self):
 
     def monitor_aws_termination_thread(self):
         while True:
+            duration = 5
             if self.termination_event.isSet():
                 break
             self.wait_ssh_up(verbose=False)
             aws_message = self.get_aws_termination_notification()
             if aws_message:
                 self.log.warning('Got spot termination notification from AWS %s' % aws_message)
                 SpotTerminationEvent(node=self, aws_message=aws_message)
-            time.sleep(5)
+                terminate_action = json.loads(aws_message)
```

Comment:
add time left to the termination into SpotTerminationEvent

Reply:
resolved